### PR TITLE
Add “In the Wild” section.

### PR DESCRIPTION
**Pull request by ljharb**
The idea is that companies and/or projects can list themselves here, just like https://github.com/airbnb/javascript#in-the-wild and https://github.com/airbnb/aerosolve/pull/103.

It's in a separate file so that PRs to it don't have to touch the main README.

### CODE REVIEW

**Comment by ljharb** [comment]
I notice https://github.com/airbnb/airpal/pull/160 as well - @natesammons-nasdaq, do you have any interest in being included on this list?

File: README.md [hunk 1]
```diff
@@ -121,6 +121,9 @@ Airpal Version | Presto Versions Tested
 ---------------|-----------------------
 0.1            | 0.77, 0.87
 
+## In the Wild
+Organizations and projects using `aerosolve` can list themselves [here](inthewild.md).
```
Comment (andykram) [inline README.md]: s/aerosolve/airpal/

File: inthewild.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Please use [pull requests](https://github.com/airbnb/aerosolve/pull/new/master) to add your organization and/or project to this document!
```
Comment (andykram) [inline inthewild.md]: s/aerosolve/airpal/

File: inthewild.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+Organizations
+----------
+ - [Airbnb](https://github.com/airbnb)
+
+Projects
+----------
```
Comment (andykram) [inline inthewild.md]: Isn't the convention for markdown file names to be constantized?

File: README.md (hunk 1 above)
Comment (ljharb) [inline README.md]: doh, thanks

File: inthewild.md (hunk 3 above)
Comment (ljharb) [inline inthewild.md]: sure

**Comment by andykram** [comment]
:+1: LGTM

**Comment by natesammons-nasdaq** [comment]
Sorry for the huge delay in replying -- we'd be happy to show up in an "In the wild" section.  If you add us, please list us as "Nasdaq" (not in all-caps like it used to be) or the lawyers will be unhappy.  Let me know if you need any more info!  Thanks.

**Comment by ljharb** [comment]
@natesammons-nasdaq Thanks! I've added you in 550ad14a589a41c95a7a82d14560f3995c419188

**Pull request merged**